include/repeated_hafnian.hpp

```diff
@@ -16,14 +16,70 @@
  * \rst
  * Contains functions for computing the hafnian using the algorithm
  * described in *From moments of sum to moments of product*,
- * `doi:10.1016/j.jmva.2007.01.013 <https://dx.doi.org/10.1016/j.jmva.2007.01.013>`__.
- * \endrst
+ * `doi:10.1016/j.jmva.2007.01.013
+ * <https://dx.doi.org/10.1016/j.jmva.2007.01.013>`__. \endrst
```

Comment:
Huh does clang-format format docstrings as well?

Reply:
it just doesn't like lines above 80 characters I think, maybe this can be changed. The workaround is just to only clang-format code and not doxygen comments

Reply:
Oh no this is fine! I quite like that it tidies docstrings :laughing: Was just curious because Python's formatter ignores them.